app/views/admin/notes/_form.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= form_with model: [:admin, note], local: true do |f| %>
+  <%= f.hidden_field :submission_id %>
+  <%= f.text_area :body %>
```

Comment:
Let's add a `required` attribute so the browser will prevent the user from submitting with empty data:

```suggestion
  <%= f.text_area :body, required: "required" %>
```

Reply:
added to PR